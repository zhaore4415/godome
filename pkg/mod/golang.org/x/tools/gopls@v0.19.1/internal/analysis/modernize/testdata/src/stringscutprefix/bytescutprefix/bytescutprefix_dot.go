package bytescutprefix

import (
	. "bytes"
)

var bss, bspre []byte

// test supported cases of pattern 1
func _() {
	if HasPrefix(bss, bspre) { // want "HasPrefix \\+ TrimPrefix can be simplified to CutPrefix"
		a := TrimPrefix(bss, bspre)
		_ = a
	}
}
