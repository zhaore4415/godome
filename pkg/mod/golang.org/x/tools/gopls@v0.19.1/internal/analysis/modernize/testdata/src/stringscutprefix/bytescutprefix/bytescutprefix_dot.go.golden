package bytescutprefix

import (
	. "bytes"
)

var bss, bspre []byte

// test supported cases of pattern 1
func _() {
	if after, ok := CutPrefix(bss, bspre); ok { // want "HasPrefix \\+ TrimPrefix can be simplified to CutPrefix"
		a := after
		_ = a
	}
}